README.md

```diff
@@ -28,3 +28,20 @@ at your option.
 Unless you explicitly state otherwise, any contribution intentionally submitted
 for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
 dual licensed as above, without any additional terms or conditions.
+
+### Run integration tests
```

Comment:
We should document the requirements of the testing script too. Things like "you'll need to have Docker set up and running".

Reply:
Done